saleor/discount/tests/test_discounts.py

```diff
@@ -579,3 +580,43 @@ def test_get_discount_translated_name_no_translations(rule_info):
 
     # then
     assert translated_name is None
+
+
+def test_unique_together_order(order):
```

Comment:
Not sure if those tests are needed, for me it seems like we are testing here if the django works properly.